### PR TITLE
Fix queryCoord init deadlock when restart

### PR DESCRIPTION
**Pull request by aoiasd**
related:https://github.com/milvus-io/milvus/issues/19165
Signed-off-by: aoiasd <zhicheng.yue@zilliz.com>

### CODE REVIEW

File: internal/querycoord/handoff_handler_test.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func TestHandoffHandlerReloadFromKV(t *testing.T) {
 	err = kv.Save(key, string(value))
 	assert.Nil(t, err)
 
-	scheduler, err := newTaskScheduler(baseCtx, meta, nil, kv, nil, func() (UniqueID, error) {
+	scheduler, err := newTaskScheduler(baseCtx, meta, nil, kv, nil, 2048, func() (UniqueID, error) {
```
Comment (yah01) [inline internal/querycoord/handoff_handler_test.go:71]: use config value

File: internal/querycoord/handoff_handler_test.go [hunk 2]
```diff
@@ -201,7 +201,7 @@ func TestHandoff(t *testing.T) {
 		ctx:              ctx,
 		cancel:           cancel,
 		client:           kv,
-		triggerTaskQueue: newTaskQueue(),
+		triggerTaskQueue: newTaskQueue(2048),
```
Comment (yah01) [inline internal/querycoord/handoff_handler_test.go:204]: same as line 71

File: internal/querycoord/query_coord.go [hunk 3]
```diff
@@ -209,7 +209,7 @@ func (qc *QueryCoord) Init() error {
 		}
 
 		// init task scheduler
-		qc.scheduler, initError = newTaskScheduler(qc.loopCtx, qc.meta, qc.cluster, qc.kvClient, qc.broker, qc.idAllocator)
+		qc.scheduler, initError = newTaskScheduler(qc.loopCtx, qc.meta, qc.cluster, qc.kvClient, qc.broker, 2048, qc.idAllocator)
```
Comment (yah01) [inline internal/querycoord/query_coord.go:212]: same as line 71

**Comment by wg1026688210** [comment]
How does this pr solve the deadlock,I didn't find any lock in goroutine

**Comment by aoiasd** [comment]
> How does this pr solve the deadlock,I didn't find any lock in goroutine

In fact, the deadlock was a logical deadlock cause by lock and gochannel. The main reason was that sum of the tasks that need recover out the range of task scheduler.So in this pr, I just

**Pull request merged**